manifests/i/IHMC/CmapTools/6.04/IHMC.CmapTools.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: IHMC.CmapTools
+PackageVersion: 6.04
+PackageLocale: en-US
+Publisher: Institute for Human & Machine Cognition
+PackageName: CmapTools

```

```suggestion
PackageName: IHMC CmapTools
```
This must match Control Panel.